sql/updates/world/master/2024_99_99_99_ragefire_chasm_instance_smart_ai.sql

```diff
@@ -0,0 +1,608 @@
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `Difficulties`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param_string`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `action_param7`, `action_param_string`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_param_string`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(211883, 1, 0, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 120026, 0, 21, 20, 0, 0, 0, NULL, 21, 20, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast spell 120026 to player'),
+(211883, 1, 1, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 101163, 0, 19, 61788, 0, 0, 0, NULL, 19, 61788, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast Spell 101163 to SI:7 Ranger'),
+(211883, 1, 2, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 120292, 0, 21, 20, 0, 0, 0, NULL, 21, 20, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast spell 120292 to player'),
+(211883, 1, 3, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 101163, 0, 19, 61680, 0, 0, 0, NULL, 19, 61680, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast Spell 101163 to Kor\'kron Scout'),
+(211884, 1, 0, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 120026, 0, 21, 20, 0, 0, 0, NULL, 21, 20, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast spell 120026 to player'),
+(211884, 1, 1, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 101163, 0, 19, 61788, 0, 0, 0, NULL, 19, 61788, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast Spell 101163 to SI:7 Ranger'),
+(211884, 1, 2, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 120292, 0, 21, 20, 0, 0, 0, NULL, 21, 20, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast spell 120292 to player'),
+(211884, 1, 3, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 101163, 0, 19, 61680, 0, 0, 0, NULL, 19, 61680, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast Spell 101163 to Kor\'kron Scout'),
+(211885, 1, 0, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 120026, 0, 21, 20, 0, 0, 0, NULL, 21, 20, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed -  Cast spell 120026 to player'),
+(211885, 1, 1, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 101163, 0, 19, 61788, 0, 0, 0, NULL, 19, 61788, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast Spell 101163 to SI:7 Ranger'),
+(211885, 1, 2, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 120292, 0, 21, 20, 0, 0, 0, NULL, 21, 20, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast spell 120292 to player'),
+(211885, 1, 3, 0, '', 70, 0, 100, 0, 2, 0, 0, 0, 0, '', 86, 101163, 0, 19, 61680, 0, 0, 0, NULL, 19, 61680, 0, 0, 0, NULL, 0, 0, 0, 0, 'Scout Cage - On State Changed - Cast Spell 101163 to Kor\'kron Scout');
```

Comment:
Maybe you can link the alliance and horde related SAI respectively (so you only need 2 conditions by go)